Index: level/test_01.level
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/level/test_01.level b/level/test_01.level
new file mode 100644
--- /dev/null	(date 1642023274928)
+++ b/level/test_01.level	(date 1642023274928)
@@ -0,0 +1,12 @@
+N: test_level_1
+S: 5 5
+T: Default
+L:
+#####
+#a  b
+# C #
+#   #
+#####
+A:
+a enemy enemy_testEnemy
+b roomlink link_test_level_01_right link_test_level_02_left
\ No newline at end of file
