Index: Roomlink.hpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Roomlink.hpp b/Roomlink.hpp
new file mode 100644
--- /dev/null	(date 1642025290288)
+++ b/Roomlink.hpp	(date 1642025290288)
@@ -0,0 +1,11 @@
+//
+// Created by tobia on 12.01.2022.
+//
+#include <string>
+
+class Roomlink{
+    std::string name;
+    int posX;
+    int posY;
+
+};
